extensions/amp-facebook/1.0/base-element.js

```diff
@@ -69,13 +69,15 @@ function parseEmbed(element) {
   const embedAs = element.getAttribute('data-embed-as');
   userAssert(
     !embedAs ||
-      ['post', 'video', 'comment', 'comments', 'like', 'page'].indexOf(
-        embedAs
-      ) !== -1,
+      ['post', 'video', 'comments', 'like', 'page'].indexOf(embedAs) !== -1,
     'Attribute data-embed-as for <amp-facebook> value is wrong, should be' +
-      ' "post", "video", "comment", "comments", "like", or "page", but was: %s',
+      ' "post", "video", "comments", "like", or "page", but was: %s',
     embedAs
   );
+  userAssert(
+    embedAs !== 'comment',
```

Comment:
This condition will never be reached since the `assert` above will error and cancel it. You should swap them in an additional PR.